hack/validate/changelog-well-formed

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+changelogFile=${1:-CHANGELOG.md}
+
+if [ ! -r "$changelogFile" ]; then
+  echo "Unable to read file $changelogFile" >&2
+  false
```

Comment:
`false exit` with status 1, but `changelog-well-formed` does not exit here

https://goo.gl/kweyAV